frontend/src/components/navigations/top-nav-bar/TopNavBar.jsx

```diff
@@ -219,17 +250,22 @@ function TopNavBar() {
 
   return (
     <Row align="middle" className="topNav">
-      <Col span={6}>
-        <UnstractLogo className="topbar-logo" />
+      <Col span={6} className="platform-switch-container">
+        {isUnstract ? (
+          <UnstractLogo className="topbar-logo" />
+        ) : (
+          <WhispererLogo className="topbar-logo" />
```

Comment:
@jagadeeswaran-zipstack Let's make this a cloud component. Here are the necessary steps to do so:

1. Move the LLM Whisperer-related icons to the cloud repo.
2. Export the icons from the `index.js` file in the cloud repo.
3. Import the icon in this component using our regular try-catch mechanism.

Reply:
Got it :+1: